vmdb/app/controllers/api_controller/tags.rb

```diff
@@ -53,23 +72,45 @@ def parse_tag_from_href(data)
                klass = collection_config[:tags][:klass].constantize
                klass.find(href.split('/').last)
              end
-      tag.present? ? tag_path_to_name(tag.name) : [nil, nil]
+      tag.present? ? tag_path_to_spec(tag.name).merge(:id => tag.id) : {}
     end
 
-    def tag_path_to_name(path)
+    def tag_path_to_spec(path)
       tag_path = (path[0..7] == TAG_NAMESPACE) ? path[8..-1] : path
       parts    = tag_path.split('/')
-      [parts[1], parts[2]]
+      {:category => parts[1], :name => parts[2]}
```

Comment:
No methods in tag.rb for this offhand, also in /managed namespace, tag path have a single / after the category, so code is fine for now.